app/helpers/host_description_helper.rb

```diff
@@ -76,7 +76,7 @@ def base_host_overview_fields(host)
     fields << { :field => [_("Host group"), link_to(host.hostgroup, hosts_path(:search => %{hostgroup_title = "#{host.hostgroup}"}))], :priority => 1000 } if host.hostgroup.present?
     fields << { :field => [_("Boot time"), (boot_time = host&.reported_data&.boot_time) ? date_time_relative(boot_time) : _('Not reported')], :priority => 1100 }
     fields << { :field => [_("Location"), link_to(host.location.title, hosts_path(:search => "location = #{host.location}"))], :priority => 1200 } if host.location.present?
```

Comment:
Would you mind fixing this line too? Locations can have the exact same problem

Reply:
Sure, thank you for noticing. 
can you explain what do you mean by cherry-picking? (:

Reply:
Cherry picking is the process we use to get fixes into older releases.

What I did was `git blame -L 79,79 app/helpers/host_description_helper.rb` which pointed me to 079723bee1. Then I used `git blame 079723bee1^ app/helpers/host_description_helper.rb` (`^` means the parent of that commit). That showed 377ff667bb as the commit that changed it last. There I could see it wasn't a problem in the old version. Then you can use `git tag --contains 377ff667bb` which pointed me to 1.24.0 as the oldest release.

With that, I knew that this affects older releases and this bugfix would be beneficial for users. It's mostly up to the core maintainers to do the actual cherry pick, but understanding the process behind it never hurts.